README.md

```diff
@@ -1,92 +1,39 @@
 # Mentorship System (Android)
 
-This is the Android application of the Mentorship System.
+| Branch | [Travis](https://travis-ci.org/) | [Codacy](https://www.codacy.com/) |
+| :---: | :---: | :---: |
+| [master](https://github.com/systers/mentorship-android/tree/master) | [![Build Status](https://travis-ci.org/systers/mentorship-android.svg?branch=master)](https://travis-ci.org/systers/mentorship-android) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/ee27e44e4ac646e0afe440173ea47823?branch=master)](https://www.codacy.com/app/m-murad/mentorship-android) |
+| [develop](https://github.com/systers/mentorship-android/tree/develop) | [![Build Status](https://travis-ci.org/systers/mentorship-android.svg?branch=develop)](https://travis-ci.org/systers/mentorship-android) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/ee27e44e4ac646e0afe440173ea47823?branch=develop)](https://www.codacy.com/app/m-murad/mentorship-android) |
 
-The backend for this project can be found on [systers/mentorship-backend](https://github.com/systers/mentorship-backend) GitHub repository.
+[Mentorship System](https://github.com/systers/mentorship-backend) is an application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period.
 
-## About
+This is the Android client for the Mentorship System.
 
-Mentorship System is an application that allows women in tech to mentor each other, on career development topics, 
-through 1:1 relations during a certain period of time.
+## Setting up the project
 
-This project was proposed as [an original project of Google Summer of Code](https://summerofcode.withgoogle.com/projects/#5331289322815488) 2018 edition, by [Isabel Costa](https://github.com/isabelcosta), that was mentored by
```

Comment:
Having the origins of the project isn't interesting for any new contributor?

Reply:
This should be in the wiki, not in `readme.md`

Even for backend, we don't have this info in `readme.md`

Reply:
I can put this on the wiki Home page. The idea is to have on the README only things related to tech and future contributions?

Reply:
Just like we have for `powerup-android` [here](https://github.com/systers/powerup-android/wiki)

Reply:
Ok got it!